docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.3"
+
+services:
+  mariadb:
+    image: lscr.io/linuxserver/mariadb:latest #
+    container_name: mariadb
+    environment:
+      - PUID=501
+      - PGID=20
+      - MYSQL_ROOT_PASSWORD=password
+      - TZ=Europe/London
+    volumes:
+      - ./.db:/config
+    ports:
+      - "5306:3306"
+    restart: always
```

Comment:
Once code stopped both containers were still running, you may want to have these as "no" if you don't need them anymore